packages/css/src/components/grid/grid.scss

```diff
@@ -25,47 +25,41 @@
 }
 
 .amsterdam-grid--gap-vertical--small {
-  row-gap: calc(var(--amsterdam-grid-gap) / 2);
+  row-gap: var(--amsterdam-grid-gap-sm);
 }
 
 .amsterdam-grid--gap-vertical--large {
-  row-gap: calc(var(--amsterdam-grid-gap) * 2);
+  row-gap: var(--amsterdam-grid-gap-lg);
 }
 
-.amsterdam-grid--padding-bottom--small {
-  padding-block-end: calc(var(--amsterdam-grid-gap) / 2);
-}
-
-.amsterdam-grid--padding-bottom--medium {
-  padding-block-end: var(--amsterdam-grid-gap);
-}
-
-.amsterdam-grid--padding-bottom--large {
-  padding-block-end: calc(var(--amsterdam-grid-gap) * 2);
-}
-
-.amsterdam-grid--padding-top--small {
-  padding-block-start: calc(var(--amsterdam-grid-gap) / 2);
+.amsterdam-grid--padding-bottom--small,
+.amsterdam-grid--padding-vertical--small {
+  padding-block-end: var(--amsterdam-grid-padding-block-sm);
 }
 
-.amsterdam-grid--padding-top--medium {
-  padding-block-start: var(--amsterdam-grid-gap);
+.amsterdam-grid--padding-bottom--medium,
+.amsterdam-grid--padding-vertical--medium {
+  padding-block-end: var(--amsterdam-grid-padding-block-md);
 }
 
-.amsterdam-grid--padding-top--large {
-  padding-block-start: calc(var(--amsterdam-grid-gap) * 2);
+.amsterdam-grid--padding-bottom--large,
+.amsterdam-grid--padding-vertical--large {
+  padding-block-end: var(--amsterdam-grid-padding-block-lg);
 }
 
+.amsterdam-grid--padding-top--small,
 .amsterdam-grid--padding-vertical--small {
-  padding-block: calc(var(--amsterdam-grid-gap) / 2);
+  padding-block-start: var(--amsterdam-grid-padding-block-sm);
 }
 
+.amsterdam-grid--padding-top--medium,
 .amsterdam-grid--padding-vertical--medium {
-  padding-block: var(--amsterdam-grid-gap);
+  padding-block-start: var(--amsterdam-grid-padding-block-md);
 }
 
+.amsterdam-grid--padding-top--large,
 .amsterdam-grid--padding-vertical--large {
-  padding-block: calc(var(--amsterdam-grid-gap) * 2);
+  padding-block-start: var(--amsterdam-grid-padding-block-lg);
 }
```

Comment:
The previous way of organising the classes results in only a couple of characters more, while improving the readability a lot imo.

(Sorry, adding a suggestion is not supported)

```
.amsterdam-grid--padding-bottom--small {
	padding-block-end: var(--amsterdam-grid-padding-block-sm);
}

.amsterdam-grid--padding-bottom--medium {
	padding-block-end: var(--amsterdam-grid-padding-block-md);
}

.amsterdam-grid--padding-bottom--large {
	padding-block-end: var(--amsterdam-grid-padding-block-lg);
}

.amsterdam-grid--padding-top--small {
	padding-block-start: var(--amsterdam-grid-padding-block-sm);
}

.amsterdam-grid--padding-top--medium {
	padding-block-start: var(--amsterdam-grid-padding-block-md);
}

.amsterdam-grid--padding-top--large {
	padding-block-start: var(--amsterdam-grid-padding-block-lg);
}

.amsterdam-grid--padding-vertical--small {
	padding-block: var(--amsterdam-grid-padding-block-sm);
}

.amsterdam-grid--padding-vertical--medium {
	padding-block: var(--amsterdam-grid-padding-block-md);
}

.amsterdam-grid--padding-vertical--large {
	padding-block: var(--amsterdam-grid-padding-block-lg);
}
```